conda_build/inspect_pkg.py

```diff
@@ -41,9 +41,10 @@ def which_package(in_prefix_path, prefix):
     only one package.
     """
     norm_ipp = in_prefix_path.replace(os.sep, '/')
+    match = re.compile(fnmatch.translate(norm_ipp)).match
```

Comment:
subtle but nice! :)

Reply:
I'm not sure why there's a need for regex here. Wouldn't `match = lambda x: x == norm_ipp` be enough?

Reply:
That's not the same, there aren't many packages that rely on having different cased filenames but they definitely exist. See my other comment.